docs/output.md

```diff
@@ -1,9 +1,5 @@
 # nf-core/cageseq: Output
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/cageseq/output](https://nf-co.re/cageseq/output)
```

Comment:
wait, shouldn't that note still be there?

Reply:
I think it changed in the newest template version:
https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/docs/output.md